src/software/utils/main_sfmTransform.cpp

```diff
@@ -521,7 +764,19 @@ int aliceVision_main(int argc, char **argv)
   }
 
   sfm::applyTransform(sfmData, S, R, t);
- 
+
+#if ALICEVISION_IS_DEFINED(ALICEVISION_HAVE_GEOGRAPHIC)
+  if (alignmentMethod == EAlignmentMethod::FROM_GPS2UTM)
+  {
+    ALICEVISION_LOG_INFO("Applying additional inversion to Y and Z to make textured mesh end up aligned correctly!");
+    Eigen::Matrix3d invYZ = Eigen::Matrix3d::Identity();
+    invYZ(1,1) = -1; invYZ(2,2) = -1;
+    Eigen::Vector3d zeroT = Eigen::Vector3d::Zero();
+
+    sfm::applyTransform(sfmData, 1.0, invYZ, zeroT);
+  }
+#endif
+
```

Comment:
I don't know why there is such inversion. This smells a lot like the infamous problem of conversion between the opengl reference frame and the computer vision reference frame.
@servantftechnicolor Maybe I'm not recalling well, but wasn't there a change in the way the obj are saved that has been introduced lately? Like to have the correct visualization there was this kind of inversion applied to all saved objs?